Gemfile

```diff
@@ -11,6 +11,7 @@ source 'https://rubygems.org' do
   gem 'rake-compiler', '>= 1', '< 2'
   gem 'rspec', '>= 3.11', '< 4'
   gem 'simplecov', '>= 0.21', '< 1', group: :test, require: false
+  gem 'benchmark-ips'
```

Comment:
Can you regenerate the gemfiles with `bundle exec appraisal install` in all ruby version then commit those changes?

Reply:
Oops. Done.